### PR TITLE
feat added vcflib/vcfbreakmulti module

### LINKED ISSUE #2133: new module: vcflib/vcfbreakmulti
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

Closes #2133 

- [x] Creating module for vcflib/vcfbreakmulti
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/vcflib/vcfbreakmulti/main.nf [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vcflib:1.0.3--hecb563c_1':
+        'quay.io/biocontainers/vcflib:1.0.3--ha025227_0' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"           , emit: versions
```
Comment (jemten) [inline modules/vcflib/vcfbreakmulti/main.nf:16]: bonus points if you align the commas :)

File: modules/vcflib/vcfbreakmulti/main.nf [hunk 2]
```diff
@@ -0,0 +1,51 @@
+
```
Comment (jemten) [inline modules/vcflib/vcfbreakmulti/main.nf:1]:
```suggestion
```

File: modules/vcflib/vcfbreakmulti/main.nf [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        $args \\
+        | bgzip -c $args2 > ${prefix}.breakmulti.vcf.gz
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        vcflib: $VERSION
+    END_VERSIONS
+    """
+
+
```
Comment (jemten) [inline modules/vcflib/vcfbreakmulti/main.nf:39]:
```suggestion
```

File: modules/vcflib/vcfbreakmulti/meta.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: Compressed VCF file
+      pattern: "*.vcf.gz"
```
Comment (jemten) [inline modules/vcflib/vcfbreakmulti/meta.yml:23]: The tool could take uncompressed also right?

**Review by jemten — COMMENTED** [review]
Looks good @Lucpen 
Some small comments from my part

File: modules/vcflib/vcfbreakmulti/meta.yml (hunk 4 above)
Comment (Lucpen) [reply modules/vcflib/vcfbreakmulti/meta.yml]: Yes, solved :)

**Review by jemten — COMMENTED** [review]
👍

File: modules/vcflib/vcfbreakmulti/meta.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+name: "vcflib_vcfbreakmulti"
+description: write your description here
```
Comment (nvnieuwk) [inline modules/vcflib/vcfbreakmulti/meta.yml:2]: Can you put a description of the `vcfbreakmulti` tool here?

**Review by nvnieuwk — CHANGES_REQUESTED** [review]

File: modules/vcflib/vcfbreakmulti/meta.yml (hunk 5 above)
Comment (Lucpen) [reply modules/vcflib/vcfbreakmulti/meta.yml:2]: Done

**Comment by nvnieuwk** [comment]
Did you manually add these files instead of using `nf-core modules create`? It seems your module wasn't added to `tests/config/pytest_modules.yml`. that's the reason why some tests aren't running

**Comment by Lucpen** [comment]
No, I didn't added manually, but I had not committed the pytest_module.yml file, I have done so now

**Comment by nvnieuwk** [comment]
> No, I didn't added manually, but I had not committed the pytest_module.yml file, I have done so now, thanks for noticing!

Oh that's possible too :p

File: tests/modules/vcflib/vcfbreakmulti/test.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+- name: vcflib vcfbreakmulti test_vcflib_vcfbreakmulti
+  command: nextflow run tests/modules/vcflib/vcfbreakmulti -entry test_vcflib_vcfbreakmulti -c tests/config/nextflow.config
+  tags:
+    - vcflib/vcfbreakmulti
+    - vcflib
+  files:
+    - path: output/vcflib/test.breakmulti.vcf.gz
+      md5sum: 666d2d970399cf219dc1dc1eadff9170
+    - path: output/vcflib/versions.yml
+      contains: "1.0.3"
```
Comment (nvnieuwk) [inline tests/modules/vcflib/vcfbreakmulti/test.yml:10]:
```suggestion
      contains: ["1.0.3"]
```

This should fix the tests :p

Comment (nvnieuwk) [reply tests/modules/vcflib/vcfbreakmulti/test.yml]: Nvm I didn't look at it carefully, you don't have to check for `versions.yml` anymore. The tests will do this automatically

File: tests/modules/vcflib/vcfbreakmulti/test.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+      contains: ["1.0.3"]
+
+- name: vcflib vcfbreakmulti stub
+  command: nextflow run ./tests/modules/vcflib/vcfbreakmulti -entry test_vcflib_vcfbreakmulti -c tests/config/nextflow.config -stub
+  tags:
+    - vcflib/vcfbreakmulti
+    - vcflib
+  files:
+    - path: output/vcflib/test.breakmulti.vcf.gz
+    - path: output/vcflib/versions.yml
```
Comment (nvnieuwk) [inline tests/modules/vcflib/vcfbreakmulti/test.yml:19]:
```suggestion
    - path: output/vcflib/test.breakmulti.vcf.gz
```

File: tests/modules/vcflib/vcfbreakmulti/test.yml [hunk 8]
```diff
@@ -0,0 +1,19 @@
+- name: vcflib vcfbreakmulti
+  command: nextflow run ./tests/modules/vcflib/vcfbreakmulti -entry test_vcflib_vcfbreakmulti -c tests/config/nextflow.config
+  tags:
+    - vcflib/vcfbreakmulti
+    - vcflib
+  files:
+    - path: output/vcflib/test.breakmulti.vcf.gz
+      md5sum: 666d2d970399cf219dc1dc1eadff9170
+    - path: output/vcflib/versions.yml
+      contains: ["1.0.3"]
```
Comment (nvnieuwk) [inline tests/modules/vcflib/vcfbreakmulti/test.yml:10]:
```suggestion
      md5sum: 666d2d970399cf219dc1dc1eadff9170
```

**Review by nvnieuwk — APPROVED** [review]
LGTM! just one last change :)

**Pull request merged**